__tests__/unit/DoubleSearch.test.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import DoubleSearch from "../../screens/DoubleSearch";
+import renderer from "react-test-renderer";
+
+describe("DoubleSearch component", () => {
+    test("renders correctly", () => {
+        const navigation = { getParam: (param, defaultValue) => {
+            return "JMSB";
```

Comment:
This was a quick fix, however we should find another alternative to test the getParam() function, because in this case we're forcing the return of the value "JMSB"